docs/clients/README.md

```diff
@@ -16,3 +16,13 @@ Light-clients enable users to interact with your application without having to d
 
 - [Command-Line interface for SDK-based blockchain](./cli.md)
 - [Service provider doc](./service-providers.md)
+
```

Comment:
I'm not really sure this doc should stay here @alexanderbez @alessio

Reply:
Still unsure about this detail

Reply:
I think this is fine for now. Maybe we want to build it as a binary from make?